src/Files.Launcher/Helpers/CloudDrivesDetector.cs

```diff
@@ -86,7 +92,7 @@ private static Task<List<CloudProvider>> DetectGenericCloudDrive()
                         {
                             CloudProviders.Mega => $"MEGA ({Path.GetFileName(syncedFolder.TrimEnd('\\'))})",
                             CloudProviders.AmazonDrive => $"Amazon Drive",
-                            CloudProviders.Nextcloud => $"Nextcloud",
+                            CloudProviders.Nextcloud => $"{ (namespaceKey?.GetValue("")!=null ? (string)namespaceKey?.GetValue(""):"Nextcloud")}",
```

Comment:
`!string.IsNullOrEmpty(namespaceKey?.GetValue(""))` might be better.

Reply:
suggested change have been applied in 1b4fe03